%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                                                 %%
%% Please do not use \input{...} to include other tex files.       %%
%% Submit your LaTeX manuscript as one .tex document.              %%
%%                                                                 %%
%% All additional figures and files should be attached             %%
%% separately and not embedded in the \TeX\ document itself.       %%
%%                                                                 %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%\documentclass[referee,sn-basic]{sn-jnl}% referee option is meant for double line spacing

%%=======================================================%%
%% to print line numbers in the margin use lineno option %%
%%=======================================================%%

%%\documentclass[lineno,sn-basic]{sn-jnl}% Basic Springer Nature Reference Style/Chemistry Reference Style

%%======================================================%%
%% to compile with pdflatex/xelatex use pdflatex option %%
%%======================================================%%
\documentclass[sn-standardnature]{sn-jnl}% Standard Nature Portfolio Reference Style
%%\documentclass[sn-standardnature]{sn-jnl}% Standard Nature Portfolio Reference Style
    \usepackage{etoolbox}
    \makeatletter
    \patchcmd{\ps@headings}%% Regular Pages Style
    {\hbox to \hsize{\hfill Springer Nature 2021 \LaTeX\ template\hfill}}
    {\hbox to \hsize{}}
    {}
    {}
    \patchcmd{\ps@titlepage}%% Opening Page Style
    {\hbox to \hsize{\hfill Springer Nature 2021 \LaTeX\ template\hfill}}
    {\hbox to \hsize{}}
    {}
    {}
    \patchcmd{\ps@headings}%% Regular Pages Style
    {\hbox to \hsize{\hfill Springer Nature 2021 \LaTeX\ template\hfill}}
    {\hbox to \hsize{}}
    {}
    {}
%%\documentclass[pdflatex,sn-basic]{sn-jnl}% Basic Springer Nature Reference Style/Chemistry Reference Style
%%<additional latex packages if required can be included here>

%%\documentclass[sn-basic]{sn-jnl}% Basic Springer Nature Reference Style/Chemistry Reference Style
%%\documentclass[pdflatex,sn-mathphys]{sn-jnl}% Math and Physical Sciences Reference Style
%%\documentclass[sn-aps]{sn-jnl}% American Physical Society (APS) Reference Style
%%\documentclass[sn-vancouver]{sn-jnl}% Vancouver Reference Style
%%\documentclass[sn-apa]{sn-jnl}% APA Reference Style
%%\documentclass[sn-chicago]{sn-jnl}% Chicago-based Humanities Reference Style
%%\documentclass[sn-standardnature]{sn-jnl}% Standard Nature Portfolio Reference Style
%%\documentclass[default]{sn-jnl}% Default
%%\documentclass[default,iicol]{sn-jnl}% Default with double column layout

%%%% Standard Packages
%%<additional latex packages if required can be included here>
%%%%

%%%%%=============================================================================%%%%
%%%%  Remarks: This template is provided to aid authors with the preparation
%%%%  of original research articles intended for submission to journals published
%%%%  by Springer Nature. The guidance has been prepared in partnership with
%%%%  production teams to conform to Springer Nature technical requirements.
%%%%  Editorial and presentation requirements differ among journal portfolios and
%%%%  research disciplines. You may find sections in this template are irrelevant
%%%%  to your work and are empowered to omit any such section if allowed by the
%%%%  journal you intend to submit to. The submission guidelines and policies
%%%%  of the journal take precedence. A detailed User Manual is available in the
%%%%  template package for technical guidance.
%%%%%=============================================================================%%%%
\usepackage{float}
\jyear{2023}%

%% as per the requirement new theorem styles can be included as shown below
\theoremstyle{thmstyleone}%
\newtheorem{theorem}{Theorem}%  meant for continuous numbers
%%\newtheorem{theorem}{Theorem}[section]% meant for sectionwise numbers
%% optional argument [theorem] produces theorem numbering sequence instead of independent numbers for Proposition
\newtheorem{proposition}[theorem]{Proposition}%
%%\newtheorem{proposition}{Proposition}% to get separate numbers for theorem and proposition etc.

\theoremstyle{thmstyletwo}%
\newtheorem{example}{Example}%
\newtheorem{remark}{Remark}%

\theoremstyle{thmstylethree}%
\newtheorem{definition}{Definition}%

\raggedbottom
%%\unnumbered% uncomment this for unnumbered level heads
%%<additional latex packages if required can be included here>

\begin{document}

\title[An Overview of the Research Development of Gene Expression Regulation]{An Overview of the Research Development of Gene Expression Regulation}

%%=============================================================%%
%% Prefix	-> \pfx{Dr}
%% GivenName	-> \fnm{Joergen W.}
%% Particle	-> \spfx{van der} -> surname prefix
%% FamilyName	-> \sur{Ploeg}
%% Suffix	-> \sfx{IV}
%% NatureName	-> \tanm{Poet Laureate} -> Title after name
%% Degrees	-> \dgr{MSc, PhD}
%% \author*[1,2]{\pfx{Dr} \fnm{Joergen W.} \spfx{van der} \sur{Ploeg} \sfx{IV} \tanm{Poet Laureate}
%%                 \dgr{MSc, PhD}}\email{iauthor@gmail.com}
%%=============================================================%%

\author*{\fnm{Shitao} \sur{Huang}}

%%==================================%%
%% sample for unstructured abstract %%
%%==================================%%

\abstract{The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. Authors are advised to check the author instructions for the journal they are submitting to for word limits and if structural elements like subheadings, citations, or equations are permitted.}

%%================================%%
%% Sample for structured abstract %%
%%================================%%

% \abstract{\textbf{Purpose:} The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.
%
% \textbf{Methods:} The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.
%
% \textbf{Results:} The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.
%
% \textbf{Conclusion:} The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.}

\keywords{keyword1, Keyword2, Keyword3, Keyword4}

%%\pacs[JEL Classification]{D8, H51}

%%\pacs[MSC Classification]{35A01, 65L10, 65L12, 65L20, 65L70}

\maketitle
\section*{Introduction}\label{sec1}

Gene expression regulation is the process of controlling which genes are expressed in a cell's DNA (for making functional products such as proteins) \cite{fletcher2023linking,burg1997regulation}. Gene expression regulation is important for several reasons. First, it conserves energy and space by turning on genes only when they are needed. Second, it enables the body to respond to changes in nutrient concentrations or environmental conditions by adjusting the metabolism of specific nutrients. Third, it is responsible for the expression of unique characteristic traits, growth and development of organisms. Fourth, it enables different cells in a multicellular organism to express very different genomes even though they contain the same DNA, leading to cell differentiation and specialization. Gene expression regulation is a necessary step in metabolic engineering because it allows the production of target products to be optimized by modifying gene expression profiles. However, gene expression regulation is also extremely complex and prone to malfunction, leading to diseases such as cancer. Along the way of exploring the process of gene expression regulation, scientists have discovered many new mechanisms, including many Nobel Prize-winning discoveries.

The significance of these discoveries is that they have advanced our understanding of how life works at all levels, from molecules to organisms to populations. They have also contributed to the improvement of human health and well-being by providing new insights into disease, treatment, prevention and diagnosis. Many of these discoveries have also opened up new areas of research and inspired generations of scientists \cite{burg1997regulation}.
This article will describe the development of research on gene expression regulation, which will highlight some important discoveries and techniques.



\section*{The discovery of genetic regulation}\label{sec2}

Gregor Mende studied the inheritance of different traits in pea plants, such as seed shape, seed color, and flower color. He crossed purebred strains of peas with different traits and observed how they were passed on to offspring. He found that each trait was controlled by a pair of factors (now called genes) that segregated during gamete formation and recombined during fertilization. He also developed the concept of dominant and recessive traits and the laws of segregation and independent mating.

Thomas Hunt Morgan studied the fruit fly (Drosophila melanogaster) and its variation in eye color, wing shape, and body size. He observed that some traits were linked and not independently assigned, suggesting that they were located on the same chromosome. He also found that some traits are sex-linked, meaning they are determined by genes on the X or Y chromosomes. He also confirmed that chromosomes are carriers of genetic information.

Oswald Avery and his colleagues Maclyn McCarty and Colin MacLeod conducted experiments on bacteria (Streptococcus pneumoniae) and their ability to convert from a harmless form to a virulent form. They isolated different molecules such as proteins, lipids, carbohydrates and DNA from the virulent bacteria and mixed them with the harmless bacteria to see which could induce conversion. They found that only DNA could do this, providing evidence that DNA is the genetic material.

François Jacob and Jacques Monod are French biologists. They were awarded the Nobel Prize in Physiology or Medicine in 1965, jointly with André Love, for their discoveries about the genetic control of enzyme and virus synthesis.

Jacob and Monod studied how bacteria such as E. coli turn certain genes on or off depending on the availability of nutrients, such as lactose or glucose. They proposed that genes are organized into units called operons, which include a promoter, an operon and one or more structural genes that encode proteins. They also proposed that there are regulatory genes that encode repressors, which are proteins that bind to the operon and prevent transcription of structural genes.

Jacob and Monod built their model based on experiments with mutants that are defective in lactose metabolism. They identified three types of mutations: constitutive, inducible and repressible. Constitutive mutants express structural genes regardless of the presence or absence of lactose. Inducible mutants express structural genes only in the presence of lactose. Repressible mutants express structural genes only when lactose is not present.

Jacob and Monod also discovered the role of messenger RNA (mRNA) as a mediator of DNA and protein synthesis. They showed that mRNA is synthesized from DNA by RNA polymerase in the promoter region and then reaches the ribosome, where it is translated into protein.

Since humans discovered the mechanism of gene expression, people have been continuously studying how to regulate gene expression for optimal purposes\cite{schug2010promoter,lee2004transcriptional,maston2006transcriptional}. Gene expression regulation is the process by which cells control the time, place, amount and type of gene expression. It can occur at different levels, such as chromatin structure, transcription initiation, transcript processing, transcript stability, translation initiation, protein modification, protein degradation, etc. Gene expression regulation enables cells to respond to environmental signals, differentiate into specialized cell types, coordinate cellular functions, maintain homeostasis, and more.


\section*{The discovery of transcription factors}\label{sec3}

Transcription factors play a crucial role in gene expression regulation by controlling the transcription of genes. Transcription is the process of making a RNA copy of a gene's DNA sequence, which carries the information needed to build a protein. Transcription factors are proteins that bind to specific regions of DNA near or far from a gene and influence its transcription rate. Depending on their function, transcription factors can be classified as activators or repressors. Activators increase transcription by recruiting RNA polymerase and other co-factors to the gene promoter. Repressors decrease transcription by blocking RNA polymerase or other activators from binding to the promoter. Additionally, transcription factors can regulate gene expression in a tissue-specific manner by interacting with enhancers and silencers. Enhancers are sequences of DNA that enhance transcription when bound by certain transcription factors. Silencers are sequences of DNA that silence transcription when bound by certain transcription factors. By using different combinations of transcription factors, enhancers and silencers, cells can fine-tune their gene expression patterns according to their needs.

Nüsslein-Volhard and Wieschaus are developmental biologists who shared the Nobel Prize in Physiology or Medicine in 1995 for their discoveries concerning “the genetic control of early embryonic development” \cite{nobel1995press,nusslein2017heidelberg,nusslein1980mutations,nobel1995summary}. They conducted a large-scale genetic screen in Drosophila melanogaster, a common fruit fly, to identify genes that are essential for the formation of body segments and organs. They found that mutations in some of these genes caused dramatic defects in embryonic development, such as missing or duplicated body parts. Their work revealed the existence and function of several key gene families, such as homeobox genes, gap genes, pair-rule genes, segment polarity genes, and hedgehog signaling pathway genes, that regulate embryonic patterning and cell differentiation. Their findings have had a profound impact on our understanding of animal development and evolution.

Mark Ptashne is a molecular biologist who pioneered the study of gene regulation by isolating and characterizing a protein called lambda repressor\cite{ptashne2004genetic,ptashne2007transcriptional,ptashne2015regulation}, which controls the genes of a virus that infects bacteria. He showed how this protein binds to specific DNA sequences and switches between two modes of viral reproduction: lytic and lysogenic. He also elucidated the molecular mechanisms of how this switch is influenced by environmental signals, such as UV light or temperature. His work on lambda repressor has been widely recognized as a paradigm for gene regulation in both prokaryotes and eukaryotes.




\section*{Epigenetic modifications and gene expression}\label{sec4}

Epigenetic modifications are changes in gene expression and other genomic functions that do not alter the DNA sequence but can affect how the DNA is read and interpreted by the cell. Epigenetic modifications can be influenced by environmental factors, such as diet, stress, or exposure to toxins, and can be reversible or heritable. Some examples of epigenetic modifications are DNA methylation, histone modification, and non-coding RNA interference. Epigenetic modifications play a role in regulating gene expression by altering the accessibility of DNA to transcription factors and other proteins that control gene activity. Epigenetic modifications can also interact with each other to produce synergistic effects on gene expression. Epigenetic regulation is important for cell differentiation, development, adaptation, and disease.

Allis and Jenuwein are two scientists who have made significant contributions to the field of epigenetics, which studies how chemical modifications on DNA and histones affect gene expression. Histones are proteins that package DNA into compact structures called nucleosomes. Allis and Jenuwein discovered that histones can be modified by various enzymes that add or remove chemical groups such as acetyl, methyl, or phosphate. These modifications alter the interactions between histones and DNA, as well as between histones and other proteins, thereby influencing gene activity. For example, Allis showed that histone acetylation is associated with gene activation, while Jenuwein identified a histone methyltransferase that can methylate specific residues in the histone tails and regulate gene silencing. Their work revealed that histone modifications are dynamic and reversible processes that play a crucial role in regulating gene expression in different cellular contexts.

Fire and Mello are two scientists who have discovered a novel mechanism of gene silencing by double-stranded RNA (dsRNA) in the nematode C. elegans. They showed that injecting dsRNA corresponding to a specific gene can trigger a sequence-specific degradation of the messenger RNA (mRNA) of that gene, resulting in a loss of function phenotype. This phenomenon was named RNA interference (RNAi) by Fire and colleagues. They also demonstrated that RNAi can be inherited by subsequent generations and can affect genes throughout the organism. Their work revealed that RNAi is a conserved biological response to dsRNA that mediates resistance to viral infections and transposable elements, as well as regulates gene expression during development and differentiation. Fire and Mello were awarded the Nobel Prize in Physiology or Medicine in 2006 for their discovery of RNAi.

In order to gain a deeper understanding of the principles of RNA interference, we will then explain in detail its principles and experimental methods.

RNAi is a powerful approach for reducing expression of endogenously expressed proteins for biological applications, or targeting the expression of pathological proteins for therapy. The main types of small RNAs involved in gene silencing are microRNAs (miRNAs) and siRNAs, which are processed from longer double-stranded RNAs by Dicer and loaded into the RNA-induced silencing complex (RISC). The RISC then guides the small RNA to its target mRNA for post-transcriptional gene silencing.

A combination of results obtained from several in vivo and in vitro experiments have gelled into a two-step mechanistic model for RNAi/PTGS. The first step, referred to as the RNAi initiating step, involves binding of the RNA nucleases to a large dsRNA and its cleavage into discrete $\approx 21-$ to $\approx 25-$siRNA. In the second step, these siRNAs join a multinuclease complex, RISC, which degrades the homologous single-stranded mRNAs.

The key insight in the process of PTGS was provided from the experiments of Baulcombe and Hamilton, who identified the product of RNA degradation as a siRNA of $\approx 25$ nucleotides of both sense and antisense polarity. siRNAs are formed and accumulate as double-stranded RNA molecules of defined chemical structures. siRNAs were detected first in plants undergoing either cosuppression or virus-induced gene silencing and were not detectable in control plants that were not silenced. siRNAs were subsequently discovered in Drosophila tissue culture cells in which RNAi was induced by introducing >500-nucleotide-long exogenous dsRNA, in Drosophila embryo extracts that were carrying out RNAi in vitro, and also in Drosophila embryos that were injected with dsRNA. Thus, the generation of siRNA (21 to 25 nucleotides) turned out to be the signature of any homology-dependent RNA-silencing event.

The siRNAs resemble breakdown products of an E. coli RNase III-like digestion. In particular, each strand of siRNA has 5' -phosphate and 3' -hydroxyl termini and 2- to 3-nucleotide 3' overhangs. Interestingly, in vitro-synthesized siRNAs can, in turn, induce specific RNA degradation when added exogenously to Drosophila cell extracts. Specific inhibition of gene expression by these siRNAs has also been observed in many invertebrate and some vertebrate systems.  Schwarz et al. provided direct biochemical evidence that the siRNAs could act as guide RNAs for cognate mRNA degradation.

RNAi is a conserved biological response to double-stranded RNA that mediates resistance to both endogenous and exogenous nucleic acids \ref{fig1}, and regulates gene expression. RNAi has a crucial role in silencing transposons and directing epigenetic modifications in the nucleus. The principles of RNAi involve four basic steps: processing of long dsRNA by Dicer into siRNAs, loading of one siRNA strand onto an Argonaute protein, target recognition by base pairing between siRNA and mRNA, and target cleavage or repression by Argonaute. The experimental methods of RNAi include various techniques for introducing dsRNA into cells or organisms, such as microinjection, electroporation, transfection, viral vectors, feeding, soaking or spraying. Additionally, there are methods for probing RNA structure and function using chemical modification, enzymatic cleavage, crosslinking or footprinting.

\begin{figure}[h]%
\centering
\includegraphics[width=0.9\textwidth]{RNAi.png}
\caption{Primary microRNAs (pri-miRNAs) are transcribed by RNA polymerases and are trimmed by the microprocessor complex (comprising Drosha and microprocessor complex subunit DCGR8) into ~70 nucleotide precursors, called pre-miRNAs(left side of the figure). miRNAs can also be processed from spliced short introns (known as mirtrons). pre-miRNAs contain a loop and usually have interspersed mismatches along the duplex. pre-miRNAs associate with exportin 5 and are exported to the cytoplasm, where a complex that contains Dicer, TAR RNA-binding protein (TRBP; also known as TARBP2) and PACT (also known as PRKRA) processes the pre-miRNAs into miRNA–miRNA* duplexes. The duplex associates with an Argonaute (AGO) protein within the precursor RNAi-induced silencing complex (pre-RISC). One strand of the duplex (the passenger strand) is removed. The mature RISC contains the guide strand, which directs the complex to the target mRNA for post-transcriptional gene silencing. The 'seed' region of an miRNA is indicated; in RNAi trigger design, the off-target potential of this sequence needs to be considered. Long dsRNAs (right side of the figure) are processed by Dicer, TRBP and PACT into small interfering RNAs (siRNAs). siRNAs are 20–24-mer RNAs and harbour 3'OH and 5'phosphate (PO4) groups, with 3' dinucleotide overhangs. Within the pre-RISC complex, an AGO protein cleaves the passenger siRNA strand. Then, the mature RISC, containing an AGO protein and the guide strand, associates with the target mRNA for cleavage. The inset shows the properties of siRNAs. The thermodynamic stability of the terminal sequences will direct strand loading. Like naturally occurring or artificially engineered miRNAs, the potential 'seed' region can be a source for miRNA-like off-target silencing. shRNA, short hairpin RNA\cite{davidson2011current}.}\label{fig1}
\end{figure}

\section*{Gene editing}\label{sec5}


Gene editing is the ability to make highly specific changes in the DNA sequence of a living organism, essentially customizing its genetic makeup. Gene editing is performed using enzymes, particularly nucleases that have been engineered to target a specific DNA sequence, where they introduce cuts into the DNA strands, enabling the removal of existing DNA and the insertion of replacement DNA. Gene editing can affect gene expression regulation, which is the process of controlling which genes in a cell’s DNA are expressed (used to make a functional product such as a protein). By altering the DNA sequence of a gene, gene editing can change its activity level, function or interactions with other molecules. Gene editing can also introduce new genes or remove unwanted genes from an organism’s genome. Gene editing has various applications in biotechnology, medicine and agriculture.

CRISPR/Cas9 technology is a powerful tool for editing the DNA of living organisms. It was developed by Emmanuelle Charpentier and Jennifer Doudna, who won the Nobel Prize in Chemistry 2020 for their discovery.

CRISPR/Cas9 technology is based on a natural system that bacteria use to defend themselves against viruses. Charpentier and Doudna figured out how to reprogram this system to cut any DNA sequence they wanted. This allows researchers to add, remove or change genes in animals, plants and microorganisms with high precision and efficiency.

CRISPR/Cas9 technology has opened up new possibilities for scientific research, biotechnology, agriculture and medicine. It has been used to create new crops, treat genetic diseases, engineer animal models and study gene function.

Gene editing is a powerful technology that can modify the DNA of living organisms, such as plants, animals, and humans. Gene editing has many potential applications, such as curing diseases, enhancing crops, creating animal models, and developing biotechnology. However, gene editing also raises ethical and social issues, such as safety, consent, justice, equity, regulation, and environmental impact.


Equations in \LaTeX\ can either be inline or on-a-line by itself (``display equations''). For
inline equations use the \verb+$...$+ commands. E.g.: The equation
$H\psi = E \psi$ is written via the command \verb+$H \psi = E \psi$+.

For display equations (with auto generated equation numbers)
one can use the equation or align environments:
\begin{equation}
\|\tilde{X}(k)\|^2 \leq\frac{\sum\limits_{i=1}^{p}\left\|\tilde{Y}_i(k)\right\|^2+\sum\limits_{j=1}^{q}\left\|\tilde{Z}_j(k)\right\|^2 }{p+q}.\label{eq1}
\end{equation}
where,
\begin{align}
D_\mu &=  \partial_\mu - ig \frac{\lambda^a}{2} A^a_\mu \nonumber \\
F^a_{\mu\nu} &= \partial_\mu A^a_\nu - \partial_\nu A^a_\mu + g f^{abc} A^b_\mu A^a_\nu \label{eq2}
\end{align}
Notice the use of \verb+\nonumber+ in the align environment at the end
of each line, except the last, so as not to produce equation numbers on
lines where no equation numbers are required. The \verb+\label{}+ command
should only be used at the last line of an align environment where
\verb+\nonumber+ is not used.
\begin{equation}
Y_\infty = \left( \frac{m}{\textrm{GeV}} \right)^{-3}
    \left[ 1 + \frac{3 \ln(m/\textrm{GeV})}{15}
    + \frac{\ln(c_2/5)}{15} \right]
\end{equation}
The class file also supports the use of \verb+\mathbb{}+, \verb+\mathscr{}+ and
\verb+\mathcal{}+ commands. As such \verb+\mathbb{R}+, \verb+\mathscr{R}+
and \verb+\mathcal{R}+ produces $\mathbb{R}$, $\mathscr{R}$ and $\mathcal{R}$
respectively (refer Subsubsection~\ref{subsubsec2}).

\section{Tables}\label{sec5}

Tables can be inserted via the normal table and tabular environment. To put
footnotes inside tables you should use \verb+\footnotetext[]{...}+ tag.
The footnote appears just below the table itself (refer Tables~\ref{tab1} and \ref{tab2}).
For the corresponding footnotemark use \verb+\footnotemark[...]+

\begin{table}[h]
\begin{center}
\begin{minipage}{174pt}
\caption{Caption text}\label{tab1}%
\begin{tabular}{@{}llll@{}}
\toprule
Column 1 & Column 2  & Column 3 & Column 4\\
\midrule
row 1    & data 1   & data 2  & data 3  \\
row 2    & data 4   & data 5\footnotemark[1]  & data 6  \\
row 3    & data 7   & data 8  & data 9\footnotemark[2]  \\
\botrule
\end{tabular}
\footnotetext{Source: This is an example of table footnote. This is an example of table footnote.}
\footnotetext[1]{Example for a first table footnote. This is an example of table footnote.}
\footnotetext[2]{Example for a second table footnote. This is an example of table footnote.}
\end{minipage}
\end{center}
\end{table}

\noindent
The input format for the above table is as follows:

%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%
\bigskip
\begin{verbatim}
\begin{table}[<placement-specifier>]
\begin{center}
\begin{minipage}{<preferred-table-width>}
\caption{<table-caption>}\label{<table-label>}%
\begin{tabular}{@{}llll@{}}
\toprule
Column 1 & Column 2 & Column 3 & Column 4\\
\midrule
row 1 & data 1 & data 2	 & data 3 \\
row 2 & data 4 & data 5\footnotemark[1] & data 6 \\
row 3 & data 7 & data 8	 & data 9\footnotemark[2]\\
\botrule
\end{tabular}
\footnotetext{Source: This is an example of table footnote.
This is an example of table footnote.}
\footnotetext[1]{Example for a first table footnote.
This is an example of table footnote.}
\footnotetext[2]{Example for a second table footnote.
This is an example of table footnote.}
\end{minipage}
\end{center}
\end{table}
\end{verbatim}
\bigskip
%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%

\begin{table}[h]
\begin{center}
\begin{minipage}{\textwidth}
\caption{Example of a lengthy table which is set to full textwidth}\label{tab2}
\begin{tabular*}{\textwidth}{@{\extracolsep{\fill}}lcccccc@{\extracolsep{\fill}}}
\toprule%
& \multicolumn{3}{@{}c@{}}{Element 1\footnotemark[1]} & \multicolumn{3}{@{}c@{}}{Element 2\footnotemark[2]} \\\cmidrule{2-4}\cmidrule{5-7}%
Project & Energy & $\sigma_{calc}$ & $\sigma_{expt}$ & Energy & $\sigma_{calc}$ & $\sigma_{expt}$ \\
\midrule
Element 3  & 990 A & 1168 & $1547\pm12$ & 780 A & 1166 & $1239\pm100$\\
Element 4  & 500 A & 961  & $922\pm10$  & 900 A & 1268 & $1092\pm40$\\
\botrule
\end{tabular*}
\footnotetext{Note: This is an example of table footnote. This is an example of table footnote this is an example of table footnote this is an example of~table footnote this is an example of table footnote.}
\footnotetext[1]{Example for a first table footnote.}
\footnotetext[2]{Example for a second table footnote.}
\end{minipage}
\end{center}
\end{table}

In case of double column layout, tables which do not fit in single column width should be set to full text width. For this, you need to use \verb+\begin{table*}+ \verb+...+ \verb+\end{table*}+ instead of \verb+\begin{table}+ \verb+...+ \verb+\end{table}+ environment. Lengthy tables which do not fit in textwidth should be set as rotated table. For this, you need to use \verb+\begin{sidewaystable}+ \verb+...+ \verb+\end{sidewaystable}+ instead of \verb+\begin{table*}+ \verb+...+ \verb+\end{table*}+ environment. This environment puts tables rotated to single column width. For tables rotated to double column width, use \verb+\begin{sidewaystable*}+ \verb+...+ \verb+\end{sidewaystable*}+.

\begin{sidewaystable}
\sidewaystablefn%
\begin{center}
\begin{minipage}{\textheight}
\caption{Tables which are too long to fit, should be written using the ``sidewaystable'' environment as shown here}\label{tab3}
\begin{tabular*}{\textheight}{@{\extracolsep{\fill}}lcccccc@{\extracolsep{\fill}}}
\toprule%
& \multicolumn{3}{@{}c@{}}{Element 1\footnotemark[1]}& \multicolumn{3}{@{}c@{}}{Element\footnotemark[2]} \\\cmidrule{2-4}\cmidrule{5-7}%
Projectile & Energy	& $\sigma_{calc}$ & $\sigma_{expt}$ & Energy & $\sigma_{calc}$ & $\sigma_{expt}$ \\
\midrule
Element 3 & 990 A & 1168 & $1547\pm12$ & 780 A & 1166 & $1239\pm100$ \\
Element 4 & 500 A & 961  & $922\pm10$  & 900 A & 1268 & $1092\pm40$ \\
Element 5 & 990 A & 1168 & $1547\pm12$ & 780 A & 1166 & $1239\pm100$ \\
Element 6 & 500 A & 961  & $922\pm10$  & 900 A & 1268 & $1092\pm40$ \\
\botrule
\end{tabular*}
\footnotetext{Note: This is an example of table footnote this is an example of table footnote this is an example of table footnote this is an example of~table footnote this is an example of table footnote.}
\footnotetext[1]{This is an example of table footnote.}
\end{minipage}
\end{center}
\end{sidewaystable}

\section{Figures}\label{sec6}

As per the \LaTeX\ standards you need to use eps images for \LaTeX\ compilation and \verb+pdf/jpg/png+ images for \verb+PDFLaTeX+ compilation. This is one of the major difference between \LaTeX\ and \verb+PDFLaTeX+. Each image should be from a single input .eps/vector image file. Avoid using subfigures. The command for inserting images for \LaTeX\ and \verb+PDFLaTeX+ can be generalized. The package used to insert images in \verb+LaTeX/PDFLaTeX+ is the graphicx package. Figures can be inserted via the normal figure environment as shown in the below example:

%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%
\bigskip
\begin{verbatim}
\begin{figure}[<placement-specifier>]
\centering
\includegraphics{<eps-file>}
\caption{<figure-caption>}\label{<figure-label>}
\end{figure}
\end{verbatim}
\bigskip
%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%



In case of double column layout, the above format puts figure captions/images to single column width. To get spanned images, we need to provide \verb+\begin{figure*}+ \verb+...+ \verb+\end{figure*}+.

For sample purpose, we have included the width of images in the optional argument of \verb+\includegraphics+ tag. Please ignore this.

\section{Algorithms, Program codes and Listings}\label{sec7}

Packages \verb+algorithm+, \verb+algorithmicx+ and \verb+algpseudocode+ are used for setting algorithms in \LaTeX\ using the format:

%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%
\bigskip
\begin{verbatim}
\begin{algorithm}
\caption{<alg-caption>}\label{<alg-label>}
\begin{algorithmic}[1]
. . .
\end{algorithmic}
\end{algorithm}
\end{verbatim}
\bigskip
%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%

You may refer above listed package documentations for more details before setting \verb+algorithm+ environment. For program codes, the ``program'' package is required and the command to be used is \verb+\begin{program}+ \verb+...+ \verb+\end{program}+. A fast exponentiation procedure:

\begin{program}
\BEGIN \\ %
  \FOR i:=1 \TO 10 \STEP 1 \DO
     |expt|(2,i); \\ |newline|() \OD %
\rcomment{Comments will be set flush to the right margin}
\WHERE
\PROC |expt|(x,n) \BODY
          z:=1;
          \DO \IF n=0 \THEN \EXIT \FI;
             \DO \IF |odd|(n) \THEN \EXIT \FI;
\COMMENT{This is a comment statement};
                n:=n/2; x:=x*x \OD;
             \{ n>0 \};
             n:=n-1; z:=z*x \OD;
          |print|(z) \ENDPROC
\END
\end{program}


\begin{algorithm}
\caption{Calculate $y = x^n$}\label{algo1}
\begin{algorithmic}[1]
\Require $n \geq 0 \vee x \neq 0$
\Ensure $y = x^n$
\State $y \Leftarrow 1$
\If{$n < 0$}\label{algln2}
        \State $X \Leftarrow 1 / x$
        \State $N \Leftarrow -n$
\Else
        \State $X \Leftarrow x$
        \State $N \Leftarrow n$
\EndIf
\While{$N \neq 0$}
        \If{$N$ is even}
            \State $X \Leftarrow X \times X$
            \State $N \Leftarrow N / 2$
        \Else[$N$ is odd]
            \State $y \Leftarrow y \times X$
            \State $N \Leftarrow N - 1$
        \EndIf
\EndWhile
\end{algorithmic}
\end{algorithm}
\bigskip
%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%

Similarly, for \verb+listings+, use the \verb+listings+ package. \verb+\begin{lstlisting}+ \verb+...+ \verb+\end{lstlisting}+ is used to set environments similar to \verb+verbatim+ environment. Refer to the \verb+lstlisting+ package documentation for more details.

%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%
\bigskip
\begin{minipage}{\hsize}%
\lstset{frame=single,framexleftmargin=-1pt,framexrightmargin=-17pt,framesep=12pt,linewidth=0.98\textwidth,language=pascal}% Set your language (you can change the language for each code-block optionally)
%%% Start your code-block
\begin{lstlisting}
for i:=maxint to 0 do
begin
{ do nothing }
end;
Write('Case insensitive ');
Write('Pascal keywords.');
\end{lstlisting}
\end{minipage}

\section{Cross referencing}\label{sec8}

Environments such as figure, table, equation and align can have a label
declared via the \verb+\label{#label}+ command. For figures and table
environments use the \verb+\label{}+ command inside or just
below the \verb+\caption{}+ command. You can then use the
\verb+\ref{#label}+ command to cross-reference them. As an example, consider
the label declared for Figure~\ref{fig1} which is
\verb+\label{fig1}+. To cross-reference it, use the command
\verb+Figure \ref{fig1}+, for which it comes up as
``Figure~\ref{fig1}''.

To reference line numbers in an algorithm, consider the label declared for the line number 2 of Algorithm~\ref{algo1} is \verb+\label{algln2}+. To cross-reference it, use the command \verb+\ref{algln2}+ for which it comes up as line~\ref{algln2} of Algorithm~\ref{algo1}.

\subsection{Details on reference citations}\label{subsec7}

Standard \LaTeX\ permits only numerical citations. To support both numerical and author-year citations this template uses \verb+natbib+ \LaTeX\ package. For style guidance please refer to the template user manual.

Here is an example for \verb+\cite{...}+: \cite{bib1}. Another example for \verb+\citep{...}+: \citep{bib2}. For author-year citation mode, \verb+\cite{...}+ prints Jones et al. (1990) and \verb+\citep{...}+ prints (Jones et al., 1990).

All cited bib entries are printed at the end of this article: \cite{bib3}, \cite{bib4}, \cite{bib5}, \cite{bib6}, \cite{bib7}, \cite{bib8}, \cite{bib9}, \cite{bib10}, \cite{bib11} and \cite{bib12}.

\section{Examples for theorem like environments}\label{sec10}

For theorem like environments, we require \verb+amsthm+ package. There are three types of predefined theorem styles exists---\verb+thmstyleone+, \verb+thmstyletwo+ and \verb+thmstylethree+

%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%
\bigskip
\begin{tabular}{|l|p{19pc}|}
\hline
\verb+thmstyleone+ & Numbered, theorem head in bold font and theorem text in italic style \\\hline
\verb+thmstyletwo+ & Numbered, theorem head in roman font and theorem text in italic style \\\hline
\verb+thmstylethree+ & Numbered, theorem head in bold font and theorem text in roman style \\\hline
\end{tabular}
\bigskip
%%=============================================%%
%% For presentation purpose, we have included  %%
%% \bigskip command. please ignore this.       %%
%%=============================================%%

For mathematics journals, theorem styles can be included as shown in the following examples:

\begin{theorem}[Theorem subhead]\label{thm1}
Example theorem text. Example theorem text. Example theorem text. Example theorem text. Example theorem text.
Example theorem text. Example theorem text. Example theorem text. Example theorem text. Example theorem text.
Example theorem text.
\end{theorem}

Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text.

\begin{proposition}
Example proposition text. Example proposition text. Example proposition text. Example proposition text. Example proposition text.
Example proposition text. Example proposition text. Example proposition text. Example proposition text. Example proposition text.
\end{proposition}

Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text.

\begin{example}
Phasellus adipiscing semper elit. Proin fermentum massa
ac quam. Sed diam turpis, molestie vitae, placerat a, molestie nec, leo. Maecenas lacinia. Nam ipsum ligula, eleifend
at, accumsan nec, suscipit a, ipsum. Morbi blandit ligula feugiat magna. Nunc eleifend consequat lorem.
\end{example}

Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text.

\begin{remark}
Phasellus adipiscing semper elit. Proin fermentum massa
ac quam. Sed diam turpis, molestie vitae, placerat a, molestie nec, leo. Maecenas lacinia. Nam ipsum ligula, eleifend
at, accumsan nec, suscipit a, ipsum. Morbi blandit ligula feugiat magna. Nunc eleifend consequat lorem.
\end{remark}

Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text.

\begin{definition}[Definition sub head]
Example definition text. Example definition text. Example definition text. Example definition text. Example definition text. Example definition text. Example definition text. Example definition text.
\end{definition}

Additionally a predefined ``proof'' environment is available: \verb+\begin{proof}+ \verb+...+ \verb+\end{proof}+. This prints a ``Proof'' head in italic font style and the ``body text'' in roman font style with an open square at the end of each proof environment.

\begin{proof}
Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text.
\end{proof}

Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text. Sample body text.

\begin{proof}[Proof of Theorem~{\upshape\ref{thm1}}]
Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text. Example for proof text.
\end{proof}

\noindent
For a quote environment, use \verb+\begin{quote}...\end{quote}+
\begin{quote}
Quoted text example. Aliquam porttitor quam a lacus. Praesent vel arcu ut tortor cursus volutpat. In vitae pede quis diam bibendum placerat. Fusce elementum
convallis neque. Sed dolor orci, scelerisque ac, dapibus nec, ultricies ut, mi. Duis nec dui quis leo sagittis commodo.
\end{quote}

Sample body text. Sample body text. Sample body text. Sample body text. Sample body text (refer Figure~\ref{fig1}). Sample body text. Sample body text. Sample body text (refer Table~\ref{tab3}).

\section{Methods}\label{sec11}

Topical subheadings are allowed. Authors must ensure that their Methods section includes adequate experimental and characterization data necessary for others in the field to reproduce their work. Authors are encouraged to include RIIDs where appropriate.

\textbf{Ethical approval declarations} (only required where applicable) Any article reporting experiment/s carried out on (i)~live vertebrate (or higher invertebrates), (ii)~humans or (iii)~human samples must include an unambiguous statement within the methods section that meets the following requirements:

\begin{enumerate}[1.]
\item Approval: a statement which confirms that all experimental protocols were approved by a named institutional and/or licensing committee. Please identify the approving body in the methods section

\item Accordance: a statement explicitly saying that the methods were carried out in accordance with the relevant guidelines and regulations

\item Informed consent (for experiments involving humans or human tissue samples): include a statement confirming that informed consent was obtained from all participants and/or their legal guardian/s
\end{enumerate}

If your manuscript includes potentially identifying patient/participant information, or if it describes human transplantation research, or if it reports results of a clinical trial then  additional information will be required. Please visit (\url{https://www.nature.com/nature-research/editorial-policies}) for Nature Portfolio journals, (\url{https://www.springer.com/gp/authors-editors/journal-author/journal-author-helpdesk/publishing-ethics/14214}) for Springer Nature journals, or (\url{https://www.biomedcentral.com/getpublished/editorial-policies\#ethics+and+consent}) for BMC.

\section{Discussion}\label{sec12}

Discussions should be brief and focused. In some disciplines use of Discussion or `Conclusion' is interchangeable. It is not mandatory to use both. Some journals prefer a section `Results and Discussion' followed by a section `Conclusion'. Please refer to Journal-level guidance for any specific requirements.

\section{Conclusion}\label{sec13}

Conclusions may be used to restate your hypothesis or research question, restate your major findings, explain the relevance and the added value of your work, highlight any limitations of your study, describe future directions for research and recommendations.

In some disciplines use of Discussion or 'Conclusion' is interchangeable. It is not mandatory to use both. Please refer to Journal-level guidance for any specific requirements.

\backmatter

\bmhead{Supplementary information}

If your article has accompanying supplementary file/s please state so here.

Authors reporting data from electrophoretic gels and blots should supply the full unprocessed scans for key as part of their Supplementary information. This may be requested by the editorial team/s if it is missing.

Please refer to Journal-level guidance for any specific requirements.

\bmhead{Acknowledgments}

Acknowledgments are not compulsory. Where included they should be brief. Grant or contribution numbers may be acknowledged.

Please refer to Journal-level guidance for any specific requirements.

\section*{Declarations}

Some journals require declarations to be submitted in a standardised format. Please check the Instructions for Authors of the journal to which you are submitting to see if you need to complete this section. If yes, your manuscript must contain the following sections under the heading `Declarations':

\begin{itemize}
\item Funding
\item Conflict of interest/Competing interests (check journal-specific guidelines for which heading to use)
\item Ethics approval
\item Consent to participate
\item Consent for publication
\item Availability of data and materials
\item Code availability
\item Authors' contributions
\end{itemize}

\noindent
If any of the sections are not relevant to your manuscript, please include the heading and write `Not applicable' for that section.

%%===================================================%%
%% For presentation purpose, we have included        %%
%% \bigskip command. please ignore this.             %%
%%===================================================%%
\bigskip
\begin{flushleft}%
Editorial Policies for:

\bigskip\noindent
Springer journals and proceedings: \url{https://www.springer.com/gp/editorial-policies}

\bigskip\noindent
Nature Portfolio journals: \url{https://www.nature.com/nature-research/editorial-policies}

\bigskip\noindent
\textit{Scientific Reports}: \url{https://www.nature.com/srep/journal-policies/editorial-policies}

\bigskip\noindent
BMC journals: \url{https://www.biomedcentral.com/getpublished/editorial-policies}
\end{flushleft}

\begin{appendices}

\section{Section title of first appendix}\label{secA1}

An appendix contains supplementary information that is not an essential part of the text itself but which may be helpful in providing a more comprehensive understanding of the research problem or it is information that is too cumbersome to be included in the body of the paper.

%%=============================================%%
%% For submissions to Nature Portfolio Journals %%
%% please use the heading ``Extended Data''.   %%
%%=============================================%%

%%=============================================================%%
%% Sample for another appendix section			       %%
%%=============================================================%%

%% \section{Example of another appendix section}\label{secA2}%
%% Appendices may be used for helpful, supporting or essential material that would otherwise
%% clutter, break up or be distracting to the text. Appendices can consist of sections, figures,
%% tables and equations etc.

\end{appendices}

%%===========================================================================================%%
%% If you are submitting to one of the Nature Portfolio journals, using the eJP submission   %%
%% system, please include the references within the manuscript file itself. You may do this  %%
%% by copying the reference list from your .bbl file, paste it into the main manuscript .tex %%
%% file, and delete the associated \verb+\bibliography+ commands.                            %%
%%===========================================================================================%%

\bibliography{sn-bibliography}% common bib file
%% if required, the content of .bbl file can be included here once bbl is generated
%%\input sn-article.bbl

%% Default %%
%%\input sn-sample-bib.tex%


\end{document}
